api/obs/api/routes/login.py

```diff
@@ -1,5 +1,7 @@
+import asyncio
 import logging
-import os
```

Comment:
``os`` was detected as a stale import by my IDE and API is working without ``os`` here for me.